src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -214,6 +214,23 @@ uct_rc_mlx5_iface_common_am_handler(uct_rc_mlx5_iface_common_t *iface,
                                       &iface->super.super.release_desc);
 }
 
+static UCS_F_ALWAYS_INLINE uint8_t
+uct_rc_mlx5_ep_fm(uct_rc_mlx5_iface_common_t *iface, uct_ib_mlx5_txwq_t *txwq)
+{
+    uint8_t fm_ce_se = MLX5_WQE_CTRL_CQ_UPDATE;
+
+    fm_ce_se |= txwq->next_fm;
+    txwq->next_fm = 0;
+    /* iface increase beat each time iface_fence requested, so, if our beat
```

Comment:
space line before

Reply:
/* a call to iface_fence increases beat, so if endpoint beat is not in sync with iface beat it means the endpoint did not post any WQE with fence flag yet */